### ISSUE TITLE
Installer will never use proxy settings

### ISSUE THREAD

**Issue by bsmartins** [ibmdb/node-ibm_db#38]
When trying install the module, download of the driver behind an enterprise firewall will always fail due to `buildHttpOptions` function always returning the default options.

```
function buildHttpOptions(installerfileURL) {
   var options = <init_value>

   ...
   exec('npm config get proxy', function(error, stdout, stderr) {

      ...
      options = <some_value>
      ...

   }
   ...   

   return options;
}
```
The above code will not affect the options because they've already been returned.

**Comment by silveirado** [comment]
Hi bsmaritns,  i fixed this issue on my fork, and submitted a pull-request #61. It is working fine for me.

**Comment by bsmartins** [comment]
Thanks @silveirado.
Any change of getting this released soon?

**Comment by bimalkjha** [comment]
Yes, it will get released soon. Please fill this form: http://goo.gl/forms/EKzg7A6HZw

**Comment by bimalkjha** [comment]
Fixed in ibm_db v0.0.15. Thanks.
